xml/System.Threading.Tasks/TaskScheduler.xml

```diff
@@ -93,11 +93,11 @@
   
  To create the example, create a WPF application project in Visual Studio and assign it a name of your choice. Then do the following:  
   
-1.  In design view, drag an <xref:System.Windows.Controls.Image> control from the **Toolbox** to the design surface. In XAML view, specify the horizontal alignment as "Left." The size does not matter because the control is be dynamically resized at run time. Accept the default name, "image".  
+1.  In design view, drag an <xref:System.Windows.Controls.Image> control from the **Toolbox** to the design surface. In XAML view, specify the horizontal alignment as "Left." The size does not matter because the control is be dynamically resized at run time. In the Name textbox of the Properties window, name the control "image".  
```

Comment:
default horizontal alignment is already left, remove that?

also from what I tested, the control should be dropped on the upper left corner, otherwise results look funny.